sentry.server.config.js

```diff
@@ -0,0 +1,24 @@
+// This file configures the initialization of Sentry on the server.
+// The config you add here will be used whenever the server handles a request.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from '@sentry/nextjs'
+
+const {
+  SENTRY_DSN,
+  SENTRY_RELEASE,
+  SENTRY_ENVIRONMENT,
+  SENTRY_DEBUG,
+  NODE_ENV,
+} = process.env
+
+Sentry.init({
+  dsn: SENTRY_DSN,
+  tracesSampleRate: 1.0,
```

Comment:
We'll probably want to turn this down at some point - suggestions in the docs were around `0.1` (i.e. 10% of requests being logged)

Reply:
Less important for bonuscalc than it is for RH